### PR TITLE
Toponaming makeElementShell

### PR DESCRIPTION
**Pull request by bgbsww**

### CODE REVIEW

File: src/Mod/Part/App/TopoShapeExpansion.cpp [hunk 1]
```diff
@@ -551,6 +554,142 @@ TopoShape::makeElementCompound(const std::vector<TopoShape>& shapes, const char*
     return *this;
 }
 
+struct MapperFill: Part::TopoShape::Mapper
+{
+    BRepFill_Generator& maker;
+    MapperFill(BRepFill_Generator& maker)
```
Comment (chennes) [inline src/Mod/Part/App/TopoShapeExpansion.cpp:560]: Can this be `explicit`?

File: src/Mod/Part/App/TopoShapeExpansion.cpp [hunk 2]
```diff
@@ -551,6 +554,142 @@ TopoShape::makeElementCompound(const std::vector<TopoShape>& shapes, const char*
...
+        : maker(maker)
+    {}
+    virtual const std::vector<TopoDS_Shape>& generated(const TopoDS_Shape& s) const override
```
Comment (chennes) [inline src/Mod/Part/App/TopoShapeExpansion.cpp:563]:
```suggestion
    const std::vector<TopoDS_Shape>& generated(const TopoDS_Shape& s) const override
```

File: src/Mod/Part/App/TopoShapeExpansion.cpp [hunk 3]
```diff
@@ -551,6 +554,142 @@ TopoShape::makeElementCompound(const std::vector<TopoShape>& shapes, const char*
...
+            if (FC_LOG_INSTANCE.isEnabled(FC_LOGLEVEL_LOG)) {
+                FC_WARN("Exception on shape mapper: " << e.GetMessageString());
+            }
+        }
+        return _res;
+    }
+};
+
+// topo naming counterpart of TopoShape::makeShell()
+TopoShape& TopoShape::makeElementShell(bool silent, const char* op)
```
Comment (chennes) [inline src/Mod/Part/App/TopoShapeExpansion.cpp:582]: I'd rather see this boolean as a `class enum` -- are there other methods in the new code that use this idea of "silent"?

Comment (bgbsww) [reply src/Mod/Part/App/TopoShapeExpansion.cpp:582]: Yes, and there is a fair history of them across the code base.  Try PartDesign::ProfileBased as one example.  I'm not sure it makes sense to change this in just one module.  Could it be a separate, code base wide change someone makes?

**Pull request merged**